CHANGELOG.md

```diff
@@ -1,5 +1,11 @@
 # Job Manager Change Log
 
+## v1.3.1 Release Notes
+
+### Fixed a bug where clicking on operation details icon will show attempt data.
+
+### Fixed a bug in the handling of scattered task shards with no start date.
```

Comment:
I think it could be valuable to describe exactly what happens when the bug triggers so that affected users can see the notes and say, "YES!! they fixed MY bug."

```suggestion
### Fixed a bug that caused Job Manager to XXXX when it attempted to process a scattered task shard with no start time.
```

Reply:
Changed – is this better?

Reply:
Yep